app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -104,14 +103,14 @@ function applyEditCallBack() {
     toggleButtons(false);
     initializeCheckboxSorting();
 
+    TinyMCE.init();
```

Comment:
This is a module that initialize TinyMCE the implementation is hidden in this module